packages/react/dialog/src/Dialog.tsx

```diff
@@ -139,23 +138,21 @@ interface DialogPortalProps extends Omit<PortalProps, 'asChild'> {
   forceMount?: true;
 }
 
-const DialogPortal = React.forwardRef<DialogPortalElement, DialogPortalProps>(
-  (props: ScopedProps<DialogPortalProps>, forwardedRef) => {
-    const { __scopeDialog, forceMount, children, ...portalProps } = props;
-    const context = useDialogContext(PORTAL_NAME, __scopeDialog);
-    return (
-      <>
-        {React.Children.map(children, (child) => (
-          <Presence present={forceMount || context.open}>
-            <UnstablePortal {...portalProps} asChild ref={forwardedRef}>
-              {child}
-            </UnstablePortal>
-          </Presence>
-        ))}
-      </>
-    );
-  }
-);
+const DialogPortal: React.FC<DialogPortalProps> = (props: ScopedProps<DialogPortalProps>) => {
+  const { __scopeDialog, forceMount, children, container } = props;
+  const context = useDialogContext(PORTAL_NAME, __scopeDialog);
+  return (
+    <>
+      {React.Children.map(children, (child) => (
+        <Presence present={forceMount || context.open}>
+          <UnstablePortal asChild container={container}>
```

Comment:
I guess that works because we know Portal only has one prop for now.
Shall we worry about it later if Portal ever takes another prop? (probably won't happen anyway right? 😀)

Reply:
yeah any new optional props we add won't be flagged 🙁  but since we know this is always an `asChild` it does seem weird for it to accept and forward props to two different elements.